src/app/user/Balances/hooks/useGetAddressTokens.ts

```diff
@@ -17,21 +17,21 @@ const getTokenFunction = (
     args: functionName === 'balanceOf' ? ([userAddress.toLowerCase()] as [Address]) : ([] as []),
   }) as const
 
-type TokenData = { result: string | bigint; error?: object }
-type TokenDataArr = [TokenData, TokenData]
+type TokenData = { result: string | bigint; error?: object }[]
 
-const buildTokenBalanceObject = (symbol: keyof typeof tokenContracts, tokenDataArr?: TokenDataArr) => {
-  const [balanceData, symbolData] = tokenDataArr ?? []
-  return {
-    symbol: symbolData?.result && !symbolData?.error ? symbolData.result : symbol,
-    contractAddress: tokenContracts[symbol],
-    balance: balanceData?.result && !balanceData?.error ? balanceData.result.toString() : '0',
-  }
-}
+const buildTokenBalanceObject = (symbol: keyof typeof tokenContracts, tokenData?: TokenData) => ({
+  symbol: tokenData ? tokenData?.[1]?.result : symbol,
+  contractAddress: tokenContracts[symbol],
+  balance: tokenData?.[0]?.result ? tokenData[0].result.toString() : '0',
+})
 
-export const useGetAddressTokens = (address: Address, chainId: number) => {
-  const { data: RBTC } = useBalance({ address, chainId })
-  const { data } = useReadContracts({
+export const useGetAddressTokens = (address: Address, chainId?: number) => {
+  const { data: rbtc, isLoading: rbtcLoading, error: rbtcError } = useBalance({ address, chainId })
```

Comment:
Why was this changed to rbtc from RBTC?

Reply:
Naming convention, I assume. All-caps should be reserved to constants.